test/test_torch.py

```diff
@@ -17161,6 +17164,7 @@ def _float_to_int_conversion_helper(self, vals, device, dtype):
     # NumPy has the same behavior.
     @onlyOnCPUAndCUDA
     @unittest.skipIf(IS_MACOS, "Test is broken on MacOS, see https://github.com/pytorch/pytorch/issues/38752")
+    @unittest.skipIF(IS_PPC, "Test is borken on PowerPC, see https://github.com/pytorch/pytorch/issues/39671")
```

Comment:
```suggestion
    @unittest.skipIf(IS_PPC, "Test is borken on PowerPC, see https://github.com/pytorch/pytorch/issues/39671")
```

Looks like we have an issue with a capital `F` here